akka-docs/src/main/paradox/java/stream/stream-integrations.md

```diff
@@ -1,514 +1 @@
-method to provide a timeout after which this Publisher should be considered canceled. The actor will be notified when
-the timeout triggers via an `ActorPublisherMessage.SubscriptionTimeoutExceeded` message and MUST then perform
-cleanup and stop itself.
-
-If the actor is stopped the stream will be completed, unless it was not already terminated with
-failure, completed or canceled.
-
-More detailed information can be found in the API documentation.
-
-This is how it can be used as input `Source` to a `Flow`:
-
-@@snip [ActorPublisherDocTest.java]($code$/java/jdocs/stream/ActorPublisherDocTest.java) { #actor-publisher-usage }
-
-You can only attach one subscriber to this publisher. Use a `Broadcast`-element or
-attach a `Sink.asPublisher(AsPublisher.WITH_FANOUT)` to enable multiple subscribers.
-
-#### ActorSubscriber
-
-@@@ warning
-
-**Deprecation warning:** `ActorSubscriber` is deprecated in favour of the vastly more
-type-safe and safe to implement `akka.stream.stage.GraphStage`. It can also
-expose a "stage actor ref" is needed to be addressed as-if an Actor.
-Custom stages implemented using `GraphStage` are also automatically fusable.
-
-To learn more about implementing custom stages using it refer to @ref:[Custom processing with GraphStage](../stream/stream-customize.md#graphstage).
-
-@@@
-
-Extend `akka.stream.actor.AbstractActorSubscriber` to make your class a stream subscriber with
-full control of stream back pressure. It will receive
-`ActorSubscriberMessage.OnNext`, `ActorSubscriberMessage.OnComplete` and `ActorSubscriberMessage.OnError`
-messages from the stream. It can also receive other, non-stream messages, in the same way as any actor.
-
-Here is an example of such an actor. It dispatches incoming jobs to child worker actors:
-
-@@snip [ActorSubscriberDocTest.java]($code$/java/jdocs/stream/ActorSubscriberDocTest.java) { #worker-pool }
-
-Subclass must define the `RequestStrategy` to control stream back pressure.
-After each incoming message the `AbstractActorSubscriber` will automatically invoke
-the `RequestStrategy.requestDemand` and propagate the returned demand to the stream.
-
- * The provided `WatermarkRequestStrategy` is a good strategy if the actor performs work itself.
- * The provided `MaxInFlightRequestStrategy` is useful if messages are queued internally or
-delegated to other actors.
- * You can also implement a custom `RequestStrategy` or call `request` manually together with
-`ZeroRequestStrategy` or some other strategy. In that case
-you must also call `request` when the actor is started or when it is ready, otherwise
-it will not receive any elements.
-
-More detailed information can be found in the API documentation.
-
-This is how it can be used as output `Sink` to a `Flow`:
-
-@@snip [ActorSubscriberDocTest.java]($code$/java/jdocs/stream/ActorSubscriberDocTest.java) { #actor-subscriber-usage }
+[](../../scala/stream/stream-integrations.md)
```

Comment:
Oh, it seems you added a link here instead of making this file a symlink to the one in the `scala` tree - could you use a symlink instead?